backend/.pre-commit-config.yaml

```diff
@@ -1,8 +1,9 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 repos:
-  - repo: https://github.com/PyCQA/flake8
-    rev: 3.9.2
+  - repo: https://github.com/psf/black
```

Comment:
Could you please add custom pre-commit to run test coverage and should also check if coverage is not < 83% and should stop from git commit.